Components/Sources/Components/Components/Suggested Edits/Image Recommendations/WKImageRecommendationsViewModel.swift

```diff
@@ -247,7 +249,8 @@ public final class WKImageRecommendationsViewModel: ObservableObject {
                     description: metadata.description,
                     descriptionURL: metadata.descriptionUrl,
                     reason: metadata.reason,
-                    wikitext: page.revisions.first?.wikitext
+                    wikitext: page.revisions.first?.wikitext,
+                    sectionNumber: Int(metadata.sectionNumber ?? String())
```

Comment:
I still don't really trust the metadata's `sectionNumber`, because it's always `null` when I inspect the API response. I think you should hardcode 0 here, because we are explicitly [fetching section 0 wikitext](https://github.com/wikimedia/wikipedia-ios/blob/main/WKData/Sources/WKData/Data%20Controllers/WKGrowthTasks/WKGrowthTasksDataController.swift#L30) in the image recommendations call.

We want this section number to stay in sync with what wikitext we're using to inject image wikitext into. I suspect that when `sectionNumber` is null, once we connect the posting to `EditSaveViewController`, you will be updating the entire article wikitext with only the intro text. If we use section 0 and pass that along into `EditSaveViewController`'s `sectionID` property, it should post the update to the correct section.

Reply:
Done!